src/sprite.cpp

```diff
@@ -207,6 +221,17 @@ template <typename T> T* from_table(T* table, int level, int number) {
     const errordata* loc_warnings = asar_getwarnings(&warn_count);
     for (int i = 0; i < warn_count; i++)
         warnings.emplace_back(loc_warnings[i].fullerrdata);
+
+    if (cfg.SymbolsType.length()) {
+        const char* symbols_contents = asar_getsymbolsfile(cfg.SymbolsType.c_str());
+        std::string symbols_path = std::string(patch_name_rel) + "." + cfg.SymbolsType;

```

Comment:
Same thing here, this can be changed to 
`std::string symbols_path{fs::path{patch_name_rel}.replace_extension(cfg.SymbolsType).generic_string()};`
to avoid double file extensions.